src/ott/solvers/nn/neuraldual.py

```diff
@@ -245,7 +313,7 @@ def __call__(  # noqa: D102
         validloader_target,
         callback=callback,
     )
-    res = self.to_dual_potentials()
+    res = self.to_dual_potentials(finetune_g=self.conjugate_solver is not None)
```

Comment:
This is no longer needed thanks to #327, please remove (also from all the other calls).